data/basics/userBalance.go

```diff
@@ -186,6 +186,13 @@ type AssetParams struct {
 	// created.
 	Total uint64 `codec:"t"`
 
+	// Decimals specifies the number of digits to display after the decimal
+	// place when displaying this asset. A value of 0 represents an asset
+	// that is not divisible, a value of 1 represents an asset divisible
+	// into tenths, and so on. This value must be between 0 and 19
+	// (inclusive).
+	Decimals uint32 `codec:"dc"`
```

Comment:
not dramatic, but why 32bits if we allow a range that is smaller than 8?

Reply:
I mentioned this in the PR, but I remembered us having issues on some platforms when there were non word-aligned fields in structs. I talked about it with Tsachi though and that seemed to only happen when there were locks in the struct.